Services/Exceptions/NoAuthorizationException.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+
+namespace Microsoft.Azure.IoTSolutions.UIConfig.Services.Exceptions
+{
+    /// <summary>
+    /// This exception is thrown when the user is not authorized to perform the action.
+    /// </summary>
+    public class NoAuthorizationException : Exception
```

Comment:
I think NotAuthorizedExecption would be a better name.  The semantics fit better with system/framework exception naming.

Reply:
I think NotAuthorizedException based on comment above makes sense. We should open a bug in the new repo so that we can fix the naming in one PR later.

Reply:
it looks very similar to HTTP Unauthorized 401 which is actually a not-authenticated error. we should use Forbidden 403 which means even we know who you are, but you don't have enough permission/authorization. I intent to keep using noun sense as other exceptions to avoid confusion. But NotAuthorizedExecption is also a good name for me.